### ISSUE TITLE
make the people who make suggestions and the people who make bug reports less stupid

### ISSUE THREAD

**Issue by Haznobrain** [Anuken/Mindustry-Suggestions#5375]
### Describe what you would like changed, and why.

I would like for people who make bug reports and suggestions a bit less stupid, because i see shit like this and it hurts my eyes.
![image](https://github.com/user-attachments/assets/552da53f-735c-43da-b413-6fb6547071bd)
![image](https://github.com/user-attachments/assets/78e1fc81-9935-4449-8a35-d220794d1ca5)

### Describe the changes you want to propose. Include possible alternatives.

It would make the community be more involved with the game.

A possible alternative if anuke doesn't have the power to make people smarter is to either:
-require proof of a person's intelligence before they can file a bug report or suggestion
-make it so if a person's suggestion/bug report gets over 5 negative reactions with no positive attention their computer explodes. Or do the same if the thing they reported wasn't a bug. Or if their suggestion was clearly ai-generated do the same too.

This would greatly improve the community feedback.

**Comment by OnyxFinger** [comment]
Least dystopian regulation system :

**Comment by SandJ** [comment]
tl;dr: go have a beer.
I believe "Make people less stupid" is on the Universal Trello just under "Eliminate poverty", "Cure cancer" and "Achieve lasting world peace".  You may have a long wait.
(a) There is a very common workaround: lower your expectations and stop caring.
(b) If that does not work for you (it doesn't for me), accept it is a design 'feature'  that we all have in one area or another.  Of the 8 billion numpties on the Earth, it just takes each one to do a little creative idiocy once a year for it to feel like a barrage of brainlessness - but we forget that time that we did that embarrassing thing a few months ago we don't like to think about.
(c) Another approach is to keep them busy.  If someone wants a new power model, suggest they write the mod.  If someone is confused by the tech tree purchases (exactly as I was), suggest they write a FAQ or Wiki pages.  That's why society has charities: to ensure those who want to change the world are kept too busy with meetings and plans and budgets and minutes to actually change anything.

**Comment by OnyxFinger** [comment]
@SandJ I am 90% sure this is a joke post, however ballsy it might be

**Comment by Haznobrain** [comment]
> @SandJ I am 90% sure this is a joke post, however ballsy it might be

this is a joke post and complaint post about the quality of community feedback at the same time

**Comment by deepspacerunner** [comment]
this was funny and a good read
anuke please add :trollface:

**Comment by guncatt** [comment]
real
